### PR TITLE
Remove duplicated 'driver' function 

### PR DESCRIPTION
**Pull request by joaobarbosa**
I was checking the implementation and I noticed that, so I just removed the duplicated code and a `;` that wasn't matching the whole file format.

Also, updated `package-*.json` and added the change to the `History.md` file (hope it's not a problem! 😄 ).

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -198,19 +198,6 @@ function Crawler(driver) {
     return crawler
   }
 
-  /**
-   * Specify the driver
-   *
-   * @param {Function} driver
-   * @return {Function|Crawler}
-   */
```
[the remark this answers is not in the document]
Comment (joaobarbosa) [reply lib/index.js]: The function I've kept already have the [its own JS doc](https://github.com/matthewmueller/x-ray-crawler/pull/13/files#diff-6d186b954a58d5bb740f73d84fe39073R133):

```js
  /**
   * Get or set the driver
   *
   * @param {Function} driver
   * @return {Function|Crawler}
   * @api public
   */
```
Is it what you're worried? Or maybe should I change the text...?

[the remark this answers is not in the document]
Comment (Ilshidur) [reply lib/index.js]: I did not see it. Nevermind :-)

**Comment by joaobarbosa** [comment]
@Ilshidur if you have an idea about which linter do you prefer to use, I can add it to the project and format all files, if you want me to. Then I can add the changes to this PR or open another one. 😃

**Review by Ilshidur — APPROVED** [review]
LGMT. Sorry for the delay.

**Comment by Ilshidur** [comment]
Released under the `2.0.5` version. Thank you for this PR !

**Pull request merged**